.changelog/2127.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
great to have an example, but not sure if this particular PR should be in the changelog as it doesn't have a customer-face feature

Reply:
Probably not. I can remove it before merging.

Reply:
agreed, lets make sure to remove before the merge